File: market/test/java/ru/yandex/market/wms/shippingsorter/configuration/TestSecurityDataProvider.java
package ru.yandex.market.wms.shippingsorter.configuration;

import java.util.HashSet;
import java.util.Set;

import org.springframework.context.annotation.Profile;
import org.springframework.stereotype.Component;

import ru.yandex.market.wms.shared.libs.authorization.SecurityDataProvider;
import ru.yandex.market.wms.shared.libs.env.conifg.Profiles;

@Profile(Profiles.TEST)
@Component
public class TestSecurityDataProvider implements SecurityDataProvider {
    @Override
    public String getUser() {
        return "TEST";
    }

    @Override
    public String getToken() {
        return "TEST_TOKEN";
    }

    @Override
    public Set<String> getRoles() {
        return new HashSet<>();
    }
}

File: market/test/kotlin/ru/yandex/market/wms/dimensionmanagement/configuration/TestSecurityDataProvider.kt
package ru.yandex.market.wms.dimensionmanagement.configuration

import org.springframework.context.annotation.Profile
import org.springframework.stereotype.Component
import ru.yandex.market.wms.shared.libs.env.conifg.Profiles

@Profile(ru.yandex.market.wms.shared.libs.env.conifg.Profiles.TEST)
@Component
class TestSecurityDataProvider : ru.yandex.market.wms.shared.libs.authorization.SecurityDataProvider {
    override fun getUser(): String {
        return "TEST"
    }

    override fun getToken(): String {
        return "TEST_TOKEN"
    }

    override fun getRoles(): HashSet<String> {
        return HashSet()
    }

}
